### PR TITLE
Distribution:

### PR DESCRIPTION
**Pull request by manolama**
- Create an all-in-one build with working Horizon Config to try out the UI.

### CODE REVIEW

File: distribution/src/resources/docker/opentsdb_all.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+  pluginLocations:
+  continueOnError: true
+  loadDefaultInstances: true
+
+# Mock Data Store Settings
+MockDataStore.register.writer: true
+
+tsd.query.tnrouter.config:
+  - sourceId: atlas
```
Comment (anonthing) [inline distribution/src/resources/docker/opentsdb_all.yaml:64]: Wanna remove this? 😁

**Review by anonthing — CHANGES_REQUESTED** [review]

File: distribution/src/resources/docker/opentsdb_all.yaml (hunk 1 above)
Comment (manolama) [reply distribution/src/resources/docker/opentsdb_all.yaml:64]: doh yeah thanks

**Comment by anonthing** [comment]
Also I think having a aura only config would help for testing out OpenTSDB. Just all inmem aura for a day or so. For later though.

**Comment by manolama** [comment]
Yeah I'll add that next and some code to fetch and populate test metrics.

**Pull request merged**